src/core/Akka.MultiNodeTestRunner/Program.cs

```diff
@@ -264,7 +265,8 @@ static void Main(string[] args)
                                     .Append($@"-Dmultinode.index={nodeTest.Node - 1} ")
                                     .Append($@"-Dmultinode.role=""{nodeTest.Role}"" ")
                                     .Append($@"-Dmultinode.listen-address={listenAddress} ")
-                                    .Append($@"-Dmultinode.listen-port={listenPort} ");
+                                    .Append($@"-Dmultinode.listen-port={listenPort} ")
+                                    .Append($@"-Dmultinode.log.level={logLevel}");
```

Comment:
So, probably, instead of passing log level to `NoteTestRunner` process, we could use this `logLevel` value to apply logs filtering on the output? In this program, there is console / test output adapter actors configuration, and we could use this log level to ask those adapters to just skip the logs with level below the one we need.

Do you think we can try that?

Reply:
@IgorFedchenko am a bit confused.

Reply:
See discussion here https://github.com/akkadotnet/akka.net/issues/4215#issuecomment-738071227

I think this PR is going to do what we need, @razvangoga was working on that already for some time (still needs to be updated according to the comment above): https://github.com/akkadotnet/akka.net/pull/4644

Reply:
It's just I do not think we need to pass log level to the child processes - it can be used in MNTR process itself for output filtering.

So the idea is not to change how the logs are logged (under which level) - but how they are redirected to the console/output from MNTR process. Because we do not case about what logs are logged by NodeTestRunner processes (they are not getting to the output), it's MNTR who is collecting logs into single place and writing them to the output.
That's why the idea was to just apply filtering on the MNTR process level. This would also allow us to not touch specs configurations or how they run in NTR processes.

Reply:
It is clearer now: the filtering should happen at the `MultiNodeTestRunner`.